contracts/BatchExchange.sol

```diff
@@ -121,6 +121,18 @@ contract BatchExchange is EpochTokenLocker {
      */
     event TradeReversion(address indexed owner, uint16 indexed orderId, uint128 executedSellAmount, uint128 executedBuyAmount);
 
+    /** @dev Event emitted for each solution that is submitted
+     */
+    event SolutionSubmission(
+        address indexed submitter,
+        uint256 utility,
+        uint256 disregardedUtility,
+        uint256 burntFees,
+        uint256 lastAuctionBurntFees,
```

Comment:
How about logging the `batchIndex` as well?

If we have the `batchIndex`, then maybe we also don't need the `lastAuctionBurntFees`, as we can reconstruct it from the other numbers?

Reply:
We don't log batchIndex in any of our events as it can be derived from the timestamp of the block that emitted the event.